backend/src/Models/Elections.ts

```diff
@@ -64,13 +64,24 @@ class ElectionsDB {
         });
     }
 
-    getElections(): Promise<Election[] | null> {
+    getElections(filter: string): Promise<Election[] | null> {
+        // When I filter in trello it adds "filter=member:arendpetercastelein,overdue:true" to the URL, I'm following the same pattern here
```

Comment:
Interesting way to do this, should allow us to easily add more filtering to the front end later.